tests-data/base64_vuln/Cargo.lock

```diff
@@ -0,0 +1,25 @@
+# This file is automatically @generated by Cargo.
```

Comment:
A couple suggestions here:

- How about putting this under `tests/support` instead of `tests-data`?
- All `cargo audit` cares about is `Cargo.lock`, so I think the rest of the app boilerplate is unnecessary.

Reply:
I moved to tests/support and streamlined, but left the absolute minimal Cargo.toml in place. It seemed to me we'd want this for updating Cargo.lock when new tests were added / just for provenance / proof Cargo.lock is the generated form we should expect.

Say it once more though and I'll kill Cargo.toml and dummy.rs.

Reply:
Seems fine now